SonarScanner.Shim/Resources.Designer.cs

```diff
@@ -1,10 +1,10 @@
 ﻿//------------------------------------------------------------------------------
 // <auto-generated>
-//     This code was generated by a tool.
-//     Runtime Version:4.0.30319.42000
+//     Ce code a été généré par un outil.
```

Comment:
@valhristov @Evangelink Shouldn't such file being excluded from Git? Is it regenerated automatically during the build?